src/applications/proxy-rewrite/proxy-rewrite-whitelist.json

```diff
@@ -59,7 +59,7 @@
     {
       "hostname": "www.healthquality.va.gov",
       "pathnameBeginning": "/",
-      "cookieOnly": true
+      "cookieOnly": false
```

Comment:
If I understand it correctly, `cookieOnly` is used by the `proxy-rewrite` application to determine if the injected assets should be activated. 

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/proxy-rewrite/proxy-rewrite-entry.jsx#L191